api/controllers/v1/dochub.ts

```diff
@@ -1,48 +1,17 @@
 import { FastlyConnector } from '../../../src/services/cdn';
 import { ConsoleLogger } from '../../../src/services/logger';
 import { CDNCreds } from '../../../src/entities/creds';
-import { APIGatewayEvent, APIGatewayProxyResult } from 'aws-lambda';
-
-export const UpsertEdgeDictionaryItem = async (event: APIGatewayEvent): Promise<APIGatewayProxyResult> => {
-  if (!event.body) {
-    return {
-      statusCode: 400,
-      body: 'event.body is null',
-    };
-  }
 
+export const UpsertEdgeDictionaryItem = async (event: any = {}): Promise<any> => {
   const body = JSON.parse(event.body);
-
   const pair = {
     key: body.source,
     value: body.target,
   };
-
-  const { FASTLY_DOCHUB_SERVICE_ID, FASTLY_DOCHUB_TOKEN, FASTLY_DOCHUB_MAP } = process.env;
-
-  if (!FASTLY_DOCHUB_SERVICE_ID) {
-    return {
-      statusCode: 500,
-      body: 'Fastly dochub service ID is not defined',
-    };
-  }
-
-  if (!FASTLY_DOCHUB_TOKEN) {
-    return {
-      statusCode: 500,
-      body: 'Fastly dochub token is not defined',
-    };
-  }
-
-  if (!FASTLY_DOCHUB_MAP) {
-    return {
-      statusCode: 500,
-      body: 'Fastly dochub map is not defined',
-    };
-  }
-
-  const creds = new CDNCreds(FASTLY_DOCHUB_SERVICE_ID, FASTLY_DOCHUB_TOKEN);
-  await new FastlyConnector(new ConsoleLogger()).upsertEdgeDictionaryItem(pair, FASTLY_DOCHUB_MAP, creds);
+  // TODO: Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
```

Comment:
minor. can we use non-null assertion operator (!) after env varialbes

Reply:
Hmm, my previous PR addressed the undefined env variables [here](https://github.com/mongodb/docs-worker-pool/blob/9991af6a332db69394e919bd4d18271b2ef1bfb1/api/controllers/v1/dochub.ts#L23). This PR might be out of date